dropbox-cleaner.sh

```diff
@@ -1,63 +1,44 @@
 #!/bin/bash
+'''References:
+http://stackoverflow.com/questions/20723868/batch-rename-dropbox-conflict-files
+https://community.spiceworks.com/how_to/111611-how-to-change-all-files-folders-to-lowercase-unix
+'''
 
-#Thx, http://stackoverflow.com/questions/20723868/batch-rename-dropbox-conflict-files
+clear
 
 #Point to where you want the script to look and where it should backup files it replaces
 folder="$(pwd)"
-backup=~/Desktop/Dropbox.backup
-
-#colors
-red='\033[0;31m'
-purple='\033[1;35m'
-NC='\033[0m' # No Color
 
-clear
+#make all folder names (including case conflict folders yes) lowercase
+rename 'y/A-Z/a-z/' *
 
 
-echo "This script will climb through the $folder tree and repair conflict files by deleting the OLDER of the the conflicted file and its counterpart"
-if [[ $1 == "replace" ]]; then
-    echo -e "${red}This is NOT a drill.${NC} The script will backup the older of the conflicted files and then delete it from the Dropbox directory."
-else
-    echo -e "${purple}This is a dry run.${NC} You'll see what files would be replaced. Run \"./Dropbox.sh replace\" to make it run for real."
-fi
-echo "Press any key to continue..."
-echo "------------------------------"
+echo "This script will climb through the $folder tree and repair conflict files"
+echo "Press a key to continue..."
 read -n 1
+echo "------------------------------"
 
 find "$folder" -type f -print0 | while read -d $'\0' file; do
-    newname=$(echo "$file" | sed 's/ (.*conflitto.*)//')
```

Comment:
maybe no spaces words are more path friendly on all systems... just "conflics" or something like that i think it's better. What do you think?

Reply:
In Dropbox and for different platforms, the conflicts are usually labelled with the `( Case Conflict )` suffix. This is always the case for computers which are integrated to the English language (I am aware conflitto means conflict in Italian, but I think the English is more universalized and it may benefit a larger number of users of this application, what do you think?

Reply:
seems correct.